util/container/README.md

```diff
@@ -8,6 +8,7 @@ software development tools for OpenTitan, as listed in the
 
 Skip this step if planning to use the pre-built container. To build in local
 mode:
+<!-- TODO: Where can one find the pre-built container mentioned above? --->
```

Comment:
I think these use to be packaged in our [releases](https://github.com/lowRISC/opentitan/releases), which are rather stale at this point.

Additionally, I believe our CI builds it too. @milesdai can confirm? 

Maybe this is something CI could also drop in the public GCS bucket along with prebuilt FPGA bitstreams? @arunthomas

Reply:
I couldn't quite find it in the releases when I downloaded it. The CI does in fact build 3 different containers (including the developer utility container referenced here), but the final image is not uploaded anywhere or made available as an artifact. As far as I can tell, you need to build the docker image yourself.

It's one of the things I want to look into for the CI later on, but for now, we may want to just omit the text about the pre-built container since the state of the Dockerfile is not entirely clear at the moment.

Reply:
Ah got it, thanks @milesdai !

So maybe just stick with the text that says how to build it.